build/ci/.azure-devops-stages.yml

```diff
@@ -90,6 +90,7 @@ stages:
   displayName: Tests - Skia
   dependsOn:
     - Setup
+    - convert_tree
```

Comment:
I'm not sure about the impact of this change on CI times.

The skia_tests stage now runs tests for both UWP and WinUI flavors, but only WinUI run will depend on convert_tree stage. So, this will delay UWP flavor test run unnecessarily. Is it not significant that we can live with it, or do we need to split skia_tests to two stages, where one stage (UWP flavor) depends only on Setup, while the other (WinUI flavor) depends on both Setup and convert_tree?

Reply:
I think the Skia tests are relatively fast compared to mobile ones, but yes, this could probably be broken down to separate stages. For now I need this to properly stabilize multiwindow, so I will merge as is and open issue for this improvement 👍